tools/jvm-options-parser/src/main/java/org/logstash/launchers/JavaVersion.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.logstash.launchers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Helper class to compare current version of JVM with a target version.
+ * Based on JavaVersion class from elasticsearch java version checker tool
+ */
+public class JavaVersion implements Comparable<JavaVersion> {
+
+    public static final JavaVersion CURRENT = parse(System.getProperty("java.specification.version"));
+    public static final JavaVersion JAVA_11 = parse("11");
+    private final List<Integer> version;
```

Comment:
minor: I would call this `versions` or `versionNumbers` (as it contains major, minot, etc.. numbers) as it is a collection.